### PR TITLE
Progress Report via WhatsApp - html2canvas integration

### PR DESCRIPTION
**Pull request by anvmn**
#335

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2022-07-19 13-08

**Comment by anvmn** [comment]
@AronNovak please review

File: client/gulpfile.js [hunk 1]
```diff
@@ -197,6 +197,17 @@ gulp.task("copy:bower", function() {
     }))
 });
 
+// Copy html2canvas.
+gulp.task("copy:html2canvas", function() {
+  // There are unused Dexie files that causes trouble for uglify later
```
Comment (AronNovak) [inline client/gulpfile.js:202]: Unused in what way? Could we delete those files eventually?

File: client/src/elm/Translate.elm [hunk 2]
```diff
@@ -13085,6 +13096,26 @@ translationSet trans =
             , kinyarwanda = Nothing
             }
 
+        SendViaWhatsAppExecutionResultFailure ->
+            { english = "Action Failed. Please try again. If problem persisits, please contact system administrator."
```
Comment (AronNovak) [inline client/src/elm/Translate.elm]:
```suggestion
            { english = "Action Failed. Please try again. If problem persists, please contact system administrator."
```

File: client/src/elm/Translate.elm [hunk 3]
```diff
@@ -13085,6 +13096,26 @@ translationSet trans =
...
+            , kinyarwanda = Nothing
+            }
+
+        SendViaWhatsAppExecutionResultPleaseWait ->
+            { english = "Please allow up to one minute for action to complete ..."
```
Comment (AronNovak) [inline client/src/elm/Translate.elm:15175]:
```suggestion
            { english = "Please allow up to one minute for action to complete..."
```

File: client/src/js/app.js [hunk 4]
```diff
@@ -948,6 +954,91 @@ elmApp.ports.sendLocalIdsForDelete.subscribe(async function(info) {
...
+        var response = await fetch(url, {
+          method: 'POST',
+          body: formData,
+          // This prevents attaching cookies to request, to prevent
+          // sending authentication cookie, as our desired
+          // authentication method is token.
+          credentials: 'omit'
+        });
+
+        if (response.ok) {
```
Comment (AronNovak) [inline client/src/js/app.js:1010]: Shouldn't we have an `else` branch for this if we check this?

File: client/src/elm/Translate.elm (hunk 3 above)
Comment (anvmn) [reply client/src/elm/Translate.elm:15175]: This phrase was removed in follow up PR, so not point in changing this.

File: client/src/js/app.js (hunk 4 above)
Comment (anvmn) [reply client/src/js/app.js:1010]: Fixed at [9b375de](https://github.com/TIP-Global-Health/eheza-app/pull/316/commits/9b375de527e8118283550529113c3506b3608c74)

File: client/gulpfile.js (hunk 1 above)
Comment (anvmn) [reply client/gulpfile.js:202]: This comment was copied from another location, where it's not relevant as well.
Removed both comments.

**Pull request merged**